framework/src/actions/SetupMeshAction.C

```diff
@@ -230,6 +230,8 @@ SetupMeshAction::act()
   // Create the mesh object and tell it to build itself
   if (_current_task == "setup_mesh")
   {
+    TIME_SECTION("SetupMeshAction::act::setup_mesh", 1, "Setting Up Mesh", true);
```

Comment:
Why the switch to all caps? I thought before that we had only the first letter capitalized

Reply:
It looks much better when they are all caps.   It's consistent through the whole framework.